src/outputs/short_form_out.py

```diff
@@ -23,20 +23,22 @@ def create_period_year(df: pd.DataFrame) -> pd.DataFrame:
 
 
```

Comment:
Also in short form output we need to filter for instance 0 only, as we've created the instance 1 and 2 and we don't want every thing in triple!